Documentation/data-collection.md

```diff
@@ -758,7 +758,7 @@ data:
     # owners: (https://github.com/orgs/stolostron/teams/search-admin, @acm-observability-search)
     #
     # acm_console_page_count:sum counts the total number of visits for each page in ACM console.
-    - '{__name__="acm_console_page_count:sum", page=~"overview-classic|overview-fleet|search|search-details|clusters|application|governance"}'
+    - '{__name__="acm_console_page_count:sum", page=~"overview-fleet|search|search-details|clusters|virtual-machines|application|governance"}'
```

Comment:
you probably want to keep the deprecated `overview-classic` label because otherwise it might be removed from the Telemeter server allow-list (need @saswatamcode @philipgough confirmation here). But a comment + TODO would be useful.

Reply:
@simonpasquier thanks for the comment. If we remove the `overview-classic` label here will we lose history? If not then we are okay not to receive any new data.